setup.sh

```diff
@@ -47,7 +47,7 @@ else
    fi
 fi
 
-pip install django>=2.1 pillow grandalf django-prepared-query appdirs
+pip install "django>=2.1" pillow grandalf "django-prepared-query" appdirs "psycopg2-binary"
```

Comment:
Looks good to me. The comments are minor, and can be fixed later if you prefer that we merge now.

Reply:
Thanks for merging this, and cleaning up some of the mess I haven't had time to address this week.